### DISCUSSION TITLE
"Fly In" osu!taiko fun mod

### DISCUSSION THREAD

**Discussion by Saraistupid** [ppy/osu#31296] [Ideas]
A fun mod for osu!taiko making random patterns/notes from the map fly in (like in Project Diva) instead of being in the play area, this [already exists in PeepoDrumKit](https://files.catbox.moe/6osmri.mp4) as its own effect option

**Comment by orwenn22** [comment]
I tried to give a shot at implementing this for fun, here is the result :

https://github.com/user-attachments/assets/bb0853c6-193f-4c5e-9c52-22e3c0610dbc

Here is my branch if anyone is interested (the code quality is probably terrible):
https://github.com/orwenn22/osu/tree/fly_in_bad
